core/common/src/main/resources/com/alipay/sofa/rpc/common/rpc-config-default.json

```diff
@@ -231,6 +231,8 @@ PS:大家也看到了，本JSON文档是支持注释的，而标准JSON是不支
   "transport.use.epoll": false,
   //默认数据包大小 8*1024*1024
   "transport.payload.max": 8388608,
+  //默认数据包大小 8*1024*1024
+  "transport.grpc.maxInboundMessageSize": 4194304,
```

Comment:
Why messageSize is 4194304 ?

Reply:
The default maxInboundMessage Size for grpc is 4M, in order to maintain consistency with the original.